x-pack/solutions/security/plugins/security_solution/public/common/store/store.ts

```diff
@@ -288,7 +289,8 @@ export const createStore = (
     ...createMiddlewares(kibana, storage),
     telemetryMiddleware,
     ...(additionalMiddleware ?? []),
-    thunk
+    thunk,
+    listenerMiddleware.middleware
```

Comment:
naming `dataviewListenerMiddleware`

Reply:
hey, I think I should just move it somewhere outside the dataview picker folder. this listener middleware thing is not exclusive to the data view picker and can be used to register listeners for other parts of the app as well.